javascript/hasOwnProperty.md

```diff
@@ -0,0 +1,42 @@
+hasOwnProperty is a normal Javascript function that takes a string argument. Returns a boolean indicating whether the object has the specified property.
+
+	obj.hasOwnProperty(prop);
+
+Method to determine whether the properties defined in object are a direct property of that object as opposed to checking its prototypal inheritance.
+
+Similarly, the 'in' operator checks to see if a particular attribute is located anywhere in the object, however, this is inclusive of its prototype chain.
+
+Direct versus Indirect
```

Comment:
Hey nice TIL. Just to add to what Dan says adding some markdown makes things more readable. Most of the [READMEs](https://github.com/codingforeveryone/READMEs/tree/master/JavaScript) example [here](https://raw.githubusercontent.com/codingforeveryone/READMEs/master/JavaScript/writable-enumerable-configurable.md) have nice markdown structure that is easy to copy into your own documents. Having a [markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) handy also helps, but it only takes a few simple bits of markdown to make a document look really nice.